detox/test/index.ios.js

```diff
@@ -22,4 +22,5 @@ class exampleIos extends example {
   }
 }
 
+console.disableYellowBox = true;
```

Comment:
@LeoNatan why is this needed? why do we all of a sudden have yellow boxes?

Reply:
@LeoNatan I strongly advise against this

Reply:
This is only on iOS on debug. I don't need them.

Reply:
RN by default has yellowboxes on iOS 12 and 13.